pkg/controllers/disruption/metrics.go

```diff
@@ -101,8 +91,8 @@ var (
 	BudgetsAllowedDisruptionsGauge = prometheus.NewGaugeVec(
```

Comment:
I think we should be updating our variable names to reflect the the name changes